projects/rocprofiler-sdk/source/lib/python/rocpd/libpyrocpd.cpp

```diff
@@ -469,12 +468,151 @@ PYBIND11_MODULE(libpyrocpd, pyrocpd)
                         auto regions = rocpd::sql_generator<rocpd::types::region>{
                             conn, select_guid_nid_pid("regions"), region_order_by};
 
+                        const std::string create_samples_view =
+                            "CREATE TEMP VIEW "
+                            "    `samples_temp` AS "
+                            "SELECT "
```

Comment:
```suggestion
                        const std::string create_samples_view = R"(
                            CREATE TEMP VIEW
                                `samples_temp` AS
                            SELECT
                            ...)";
```
Can you use R"(...)" instead of these multiple quotes?

Reply:
Done